api/pkg/backend/service.go

```diff
@@ -447,3 +460,342 @@ func (s *APIService) listStorageType(ctx context.Context, request *restful.Reque
 	storageTypes, err := s.backendClient.ListType(ctx, listTypeRequest)
 	return storageTypes, err
 }
+
+//tiering functions
+func (s *APIService) CreateTier(request *restful.Request, response *restful.Response) {
+	log.Info("Received request for creating tier.")
+	tier := &backend.Tier{}
+	body := ReadBody(request)
```

Comment:
why request.ReadEntity() not be used?

Reply:
to validate schema unmarshal is used instead of ReadEntity